apps/python/web/routers/state.py

```
@@ -0,0 +1,63 @@
+"""GET / PUT /api/state — ``~/.ai-agent/state.json`` の読み書き。
+
+``auth_mode`` 専用の ``/api/auth/mode`` と違い、こちらは state.json 全体を返す
+／部分更新できる汎用エンドポイント。オンボーディングウィザードが
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `／` (FULLWIDTH SOLIDUS). Did you mean `/` (SOLIDUS)?

(RUF002)